core/src/main/java/io/paradoxical/cassieq/ServiceApplication.java

```diff
@@ -178,26 +170,7 @@ private void configureAdminSwagger(DropwizardResourceConfig resourceConfig) {
     }
 
     private void configureAuth(final ServiceConfiguration serviceConfiguration, final Environment environment) {
-
-        final Injector injector = getGuiceBundleProvider().getBundle().getInjector();
-
-        final Key<Authenticator<AuthorizedRequestCredentials, AccountPrincipal>> authenticatorKey = Key.get(new TypeLiteral<Authenticator<AuthorizedRequestCredentials, AccountPrincipal>>() {});
-
-        final SignedRequestAuthFilter<AccountPrincipal> authFilter =
-                SignedRequestAuthFilter.<AccountPrincipal>builder()
-                        .accountNamePathParameter("accountName")
-                        .setAuthenticator(injector.getInstance(authenticatorKey))
-                        .setPrefix("Signed")
-                        .setAuthorizer((principal, role) -> true)
-                        .setUnauthorizedHandler((prefix, realm) -> Response.status(Response.Status.UNAUTHORIZED).build())
-                        .buildAuthFilter();
-
-
-        final ChainedAuthFilter<AuthorizedRequestCredentials, AccountPrincipal> authFilterChain = new ChainedAuthFilter<>(Arrays.asList(
-                authFilter));
-
-
-        environment.jersey().register(authFilter);
+        environment.jersey().register(AuthLevelDynamicFeature.class);
```

Comment:
we probably don't need this registration of we use `@Provider` and move it to the discoverable class. (then guice will instantiate it too.)